rom/dev/src/flow/cold_reset/dice.rs

```diff
@@ -32,12 +32,21 @@ pub struct DiceInput<'a> {
 /// DICE Layer Output
 #[derive(Debug, Zeroize)]
 pub struct DiceOutput {
-    /// Subject key pair for this layer
-    pub subj_key_pair: Ecc384KeyPair,
+    /// ECC Subject key pair for this layer
+    pub ecc_subj_key_pair: Ecc384KeyPair,
 
-    /// Subject Serial Number
-    pub subj_sn: [u8; 64],
+    /// ECC Subject Serial Number
+    pub ecc_subj_sn: [u8; 64],
 
-    /// Subject Key Identifier
-    pub subj_key_id: [u8; 20],
+    /// ECC Subject Key Identifier
+    pub ecc_subj_key_id: [u8; 20],
+
+    /// MLDSA Subject key pair for this layer
+    pub mldsa_subj_key_pair: MlDsaKeyPair,
```

Comment:
I gather that code needed to generate certs for these keys will come in a follow-up PR.